tests/test-requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest
 pytest-timeout ~= 1.4.2
 hypothesis
 requests_mock
+qiskit-ibm-provider ~= 0.6.0
```

Comment:
Why override the qiskit versions already required by pytket-qiskit?

Reply:
Installing pytket-qiskit doesn't install `qiskit-ibm-provider`. This is the newer client that we are using instead of the deprecated one.

It needs to be added to `setup.py` though. I can do that in this PR